health/notifications/alarm-notify.sh.in

```diff
@@ -794,6 +804,107 @@ send_email() {
 	return 1
 }
 
+# -----------------------------------------------------------------------------
+# hangouts sender
+
+send_hangouts() {
+	local webhook="${1}" channels="${2}" httpcode sent=0 channel color payload
+
+	[ "${SEND_HANGOUTS}" != "YES" ] && return 1
+
+	case "${status}" in
+	WARNING) color="#ffa700" ;;
+	CRITICAL) color="#d62d20" ;;
+	CLEAR) color="#008744" ;;
+	*) color="#777777" ;;
+	esac
+
+	for channel in ${channels}; do
+	payload="$(
+		cat <<EOF
```

Comment:
I would suggest just putting the `cat <<EOF` on the previous line.  That way, you can reduce the indentation by one level for the here document and make things a bit easier to read.

Reply:
OK no problem. just to point out that other notifications are like this.
I can change them to if you permit.

Reply:
I actually hadn't remembered the other notifications did things this way, in that case, just keep it the way you have it so that it's consistent with everything else.